spec/factories/host.rb

```diff
@@ -2,7 +2,7 @@
   factory :host do
     sequence(:name)     { |n| "host_#{seq_padded_for_sorting(n)}" }
     sequence(:hostname) { |n| "host_#{seq_padded_for_sorting(n)}" }
-    vmm_vendor          "vmware"
+    vmm_vendor_display  "VMware"
```

Comment:
Yeah, me too. I wasn't sure what I might break, so I went conservatively with updating most things to use the method that had the same behavior as before. I can remove the setter (and update accordingly) if you think that wouldn't cause any headaches.